.devcontainer/Dockerfile

```diff
@@ -1,7 +1,10 @@
 # [Choice] Python version (use -bullseye variants on local arm64/Apple Silicon): 3, 3.10, 3.9, 3.8, 3.7, 3.6, 3-bullseye, 3.10-bullseye, 3.9-bullseye, 3.8-bullseye, 3.7-bullseye, 3.6-bullseye, 3-buster, 3.10-buster, 3.9-buster, 3.8-buster, 3.7-buster, 3.6-buster
+# Declare the build argument VARIANT in the global scope, can be overwrite with devcontainer.json build args
 ARG VARIANT=3.12
 FROM mcr.microsoft.com/vscode/devcontainers/python:${VARIANT}-bullseye
 
+# Consume the build argument in the build stage from global scope
+ARG VARIANT
```

Comment:
why is needed ?

Reply:
the first ARG only available before `FROM` command, once it switch to this image, it works in a sub build stage1 and the global ARG not available, so use the line7 to inherit from global scope, and it will available in this sub build stage.
so after this line, if you use another `FROM ` command, it also not available in another sub stage2.